data/hai missions.txt

```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
+					decline
+			`Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer, "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`
+			`"Captain <first> <last>. Can you show me your ship so I can look for jobs it's better suited for?"`
+			`She hesitates a bit, but leads you to her ship: you were expecting a small Hai ship, but to your surprise she she shows you a rather old-looking Heavy Shuttle docked near your own ship.`
+			`"You took a loan to buy a human ship?" you ask.`
+			`She looks at the ground, as if ashamed of it, "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult but I managed to convince him to sell it to me instead of going to the shipyard office, I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`
+			choice
+				`(Continue.)`
+			`Turns out this isn't as easy as you thought it'd be, the spaceport is very empty, save for the occasional worker or drunken crewmember, plus you see some workers pointing at Nanachi and murmuring, it seems her inability to get a job has given her a bad reputation here. Eventually, as you two begin scouting the edge of the docking area, you spot a troubled merchant near several crates, you approach him and he explains that his ship had some malfunctions and he cannot deliver this advanced surgical equipment to Cloudfire.`
+			`"This is the worse! One jump away from the planet and the navigation system suddenly decides to go on strike! There's no way I can make it before the deadline!"`
+			`"My friend Nanachi here could deliver it for you. We were heading there anyway, a few extra crates won't make a difference," you say, trying to sound as convincing as possible.`
+			`"Look kid, Unfettered attacks make it so they need a lot of medical supplies in Cloudfire, and this here is the best medical equipment Hai space has to offer, these few crates could save many military lives and I don't think I should be trusting something like this to a beginner," he says.`
+			`If you were the client for this job you certainly wouldn't want a beginner going through such a dangerous territory with such valuable cargo, but he seems desperate, and seeing as the spaceport is deserted he doesn't have many other captains to look for help, perhaps you could convince him.`
+			choice
+				`"Beginner or not, I'll be escorting her there. I'm pretty confident on my skills and my ship is nothing to scoff at, so I'll keep her, and the cargo, safe."`
+					goto safe
+				`"Don't worry, the planet is one jump away, what could possibly go wrong?"`
+					goto wrong
+				"(Make up stories about Nanachi's experience.)"
```

Comment:
Always use backticks, not just when what's inside them requires quotation marks.